.github/workflows/continuous-integration.yml

```
@@ -127,3 +127,11 @@ jobs:
           else
             echo -e "\033[PHPUnit execution skipped.\033[0m"
           fi
+      - name: "Jest"
+        run: |
+          if [[ -f "node_modules/.bin/jest" && -f "jest.config.js" && -d "tests/js" ]]; then
+            echo -e "\033[0;33mExecuting Jest...\033[0m"
+            NODE_OPTIONS="--experimental-vm-modules" node_modules/.bin/jest --colors --forceExit tests/js --verbose
```

- path can be configured by the `testMatch` entry in config file
- verbosity can be configured by the `verbose` entry in config file
```suggestion
            NODE_OPTIONS="--experimental-vm-modules" node_modules/.bin/jest --colors --forceExit
```